buildSrc/src/main/kotlin/io/spine/internal/Repositories.kt

```diff
@@ -0,0 +1,93 @@
+package io.spine.internal
+
+import org.gradle.api.artifacts.dsl.RepositoryHandler
+import org.gradle.api.artifacts.repositories.MavenArtifactRepository
+import java.net.URI
+
+
+/**
+ * Password credentials for a Maven repository.
+ */
+data class Credentials(
+    val username: String?,
+    val password: String?
+)
+
+/**
+ * Defines names of additional repositories commonly used in the Spine SDK projects.
+ */
+private object Repos {
+
+    const val sonatypeSnapshots = "https://oss.sonatype.org/content/repositories/snapshots"
+}
+
+/**
+ * A scrambled version of PAT generated with the only "read:packages" scope.
+ *
+ * The scrambling around PAT is necessary because GitHub analyzes commits for the presence
+ * of tokens and invalidates them.
+ *
+ * @see <a href="https://github.com/orgs/community/discussions/25629">
+ *     How to make GitHub packages to the public</a>
+ */
+object Pat {
+    private const val shade = "_phg->8YlN->MFRA->gxIk->HVkm->eO6g->FqHJ->z8MS->H4zC->ZEPq"
```

Comment:
Where did we get this value from? Is it expected to be changed anytime soon?

In general, this piece requires even more explanation, since that is an "example" project.